docs/intro.md

```diff
@@ -227,6 +231,107 @@ Please refer to the [project
 documentation](https://ogmios.dev/getting-started/building/#-documentation) for details on how to
 run and configure ogmios.
 
+### Configuration
+Partner chains are configured through a combination of configuration files and environment
+variables. On top of the files outlined below, the command line wizard provides a convenient layer
+for initializing configurations.
+
+#### `chain-spec.json`
+The `chain-spec.json` file combines generic Substrate configuration values with partner chain
+specific settings. All partner chain specific settings are stored below
+`.genesis.runtimeGenesis.config`:
+
+##### `aura`
```

Comment:
> 1. All fields in chain-spec.json and their meaning
and
> All partner chain specific settings are stored below

Why? Grandpa is not our business. Aura isn't strict requirement as well. `sudo` ? I think we don't have to document anything that is not coming from our toolkit.